Tests/UnitTests/Mocks/MockStoreKit2TransactionFetcher.swift

```diff
@@ -55,6 +56,11 @@ final class MockStoreKit2TransactionFetcher: StoreKit2TransactionFetcherType {
         }
     }
 
+    var receivedRefreshPolicy: AppTransactionRefreshPolicy? {
+        get { return self._receivedRefreshPolicy.value }
+        set { self._receivedRefreshPolicy.value = newValue }
+    }
```

Comment:
Added a little helper in the mock so we can check the received refresh policy in different tests.